src/cljs/proton/layers/core/keybindings.cljs

```diff
@@ -162,6 +162,9 @@
              :n {:title "line numbers"
                  :target (fn [atom] (.getView (.-views atom) (.getActiveTextEditor (.-workspace atom))))
                  :action "editor:toggle-line-numbers"}
+             :h {:category "highlight"
+                 :h {:title "highlight current line"
```

Comment:
well just having it under a single `h` would probably be fine and mostly match `spacemacs` binding. I wasn't sure how closely you were wanting to mimic it.